lms/static/sass/discussion/_discussion.scss

```diff
@@ -241,20 +228,17 @@ body.discussion {
 
   .discussion-column {
     @include float(right);
-    @include box-sizing(border-box);
+    box-sizing: border-box;
+    padding-left: ($baseline/2);
```

Comment:
@clrux This is useful as I wasn't aware that this was necessary for RTL. There are a large number of pre-existing usages of the direct styles through both the forums sass and all the rest of the LMS. Is this something we have to start moving to everywhere?